build.gradle.kts

```diff
@@ -114,6 +114,7 @@ dependencies {
     implementation("com.itextpdf:itextpdf:5.5.13.3")
     implementation("com.itextpdf.tool:xmlworker:5.5.13.3")
     implementation("net.sourceforge.htmlcleaner:htmlcleaner:2.29")
+    implementation("org.hibernate.validator:hibernate-validator:8.0.1.Final")
```

Comment:
I think this should be `providedCompile("org.hibernate.validator:hibernate-validator:8.0.1.Final")` instead of `implementation`? Because I think it is already provided by WildFly right? With an implementation it is packaged into our WAR which is then deployed into WildFly which effectively means that we get two separate Hibernate Validator JARs in WildFly. This can cause problems in future, e.g. when the versions go out of sync. I think it's better to use the one provided. Also to keep our WAR & ZAC Docker Image smaller.

Reply:
Is this dependency even needed at all? Because the `javax.validation` API is already available without it right? This is just the implementation and I don't think it is needed to add this? Depending on how we wish to test it maybe but in our case I think it's not needed? Or did you find you needed to add it?